docs/Changelog.md

```diff
@@ -4,6 +4,7 @@
 -------------------------
 * [#4561](https://github.com/TouK/nussknacker/pull/4561) Improvement: Users are not requrired at OAuth2 config file
 * [#4492](https://github.com/TouK/nussknacker/pull/4492) Allow to test fragments using ad hoc testing. 
+* [#4574](https://github.com/TouK/nussknacker/pull/4574) Improvements: Additional methods at KafkaClient in kafka-test-utils
```

Comment:
I think that we should add a migration guide entry as well. I see a lot of incompatible changes - for example `RichKafkaConsumer.consume` behaves differently